src/services/vendorEscrowSlashingService.ts

```
@@ -0,0 +1,232 @@
+      slashedAmount: calc.slashAmount,
+      reasonNotes,
+      loggedByUserId,
+      loggedAt: now,
+    };
+
+    contract.slashedAmount = calc.slashAmount;
+    contract.organizerRefundAmount = calc.slashAmount;
+    contract.netVendorPayout = calc.netVendorPayout;
+    contract.breachCount += 1;
+    contract.status =
+      calc.slashPercentage >= 100 ? "FULLY_SLASHED" : calc.slashPercentage > 0 ? "PARTIALLY_SLASHED" : "FUNDED_IN_ESCROW";
```

ESLint

[error] Replace `·?·"FULLY_SLASHED"·:·calc.slashPercentage·>·0·?·"PARTIALLY_SLASHED"` with `⏎········?·"FULLY_SLASHED"⏎········:·calc.slashPercentage·>·0⏎··········?·"PARTIALLY_SLASHED"⏎·········`

(prettier/prettier)